docs/source/releases.md

```diff
@@ -46,10 +46,10 @@ GA images aren't build from scratch but rather promoted from an existing release
 
 Support matrix table shows the version requirements for a particular **scylla-operator** version. Be sure to match these requirements, otherwise some functionality will not work.
 
-|                    | v1.7        | v1.6        | v1.5        | v1.4        | v1.3        | v1.2        | v1.1        | v1.0       |
-| :----------------: | :---------: | :---------: | :---------: | :---------: | :---------: | :---------: | :---------: | :--------: |
-| Kubernetes         | `>=1.20`    | `>=1.19.10` | `>=1.19.10` | `>=1.19.10` | `>=1.19`    | `>=1.19`    | `>=1.11`    | `>=1.11`   |
-| Scylla OS          | `>=4.3`     | `>=4.3`     | `>=4.3`     | `>=4.3`     | `>=4.2`     | `>=4.2`     | `>=4.0`     | `>=4.0`    |
-| Scylla Enterprise  | `>=2021.1`  | `>=2021.1`  | `>=2021.1`  | `>=2021.1`  | `>=2020.1`  | `>=2020.1`  | `>=2020.1`  | `>=2020.1` |
-| Scylla Manager     | `>=2.2`     | `>=2.2`     | `>=2.2`     | `>=2.2`     | `>=2.2`     | `>=2.2`     | `>=2.2`     | `>=2.2`    |
-| Scylla Monitoring  | `>=3.0`     | `>=3.0`     | `>=1.0`     | `>=1.0`     | `>=1.0`     | `>=1.0`     | `>=1.0`     | `>=1.0`    |
+|                   |    v1.8    |    v1.7    |    v1.6     |    v1.5     |    v1.4     |    v1.3    |    v1.2    |    v1.1    |    v1.0    |
+|:-----------------:|:----------:|:----------:|:-----------:|:-----------:|:-----------:|:----------:|:----------:|:----------:|:----------:|
+|    Kubernetes     |  `>=1.21`  |  `>=1.20`  | `>=1.19.10` | `>=1.19.10` | `>=1.19.10` |  `>=1.19`  |  `>=1.19`  |  `>=1.11`  |  `>=1.11`  |
+|     Scylla OS     |  `>=4.3`   |  `>=4.3`   |   `>=4.3`   |   `>=4.3`   |   `>=4.3`   |  `>=4.2`   |  `>=4.2`   |  `>=4.0`   |  `>=4.0`   |
+| Scylla Enterprise | `>=2021.1` | `>=2021.1` | `>=2021.1`  | `>=2021.1`  | `>=2021.1`  | `>=2020.1` | `>=2020.1` | `>=2020.1` | `>=2020.1` |
+|  Scylla Manager   |  `>=2.2`   |  `>=2.2`   |   `>=2.2`   |   `>=2.2`   |   `>=2.2`   |  `>=2.2`   |  `>=2.2`   |  `>=2.2`   |  `>=2.2`   |
+| Scylla Monitoring |  `>=3.0`   |  `>=3.0`   |   `>=3.0`   |   `>=1.0`   |   `>=1.0`   |  `>=1.0`   |  `>=1.0`   |  `>=1.0`   |  `>=1.0`   |
```

Comment:
ATM we don't really support any of those, it's broken with 3.0 anyway. With #1001 users will be instructed to download `branch-4.0` directly since there was a required backport which is missing from release 4.0. So I suppose 4.0 would be the most fitting, although you'd get errors if you followed our docs with any release, including 4.0.